rust_src/remacs-sys/lib.rs

```diff
@@ -823,6 +823,13 @@ pub struct Lisp_Process {
     /// Queue for storing waiting writes.
     pub write_queue: Lisp_Object,
 
+    /// Process ID.  A positive value is a child process ID.
+    /// Zero is for pseudo-processes such as network or serial connections,
+    /// or for processes that have not been fully created yet.
+    /// -1 is for a process that was not created successfully.
+    /// -2 is for a pty with no process, e.g., for GDB.  */
+    pub pid: libc::pid_t,
```

Comment:
This will not work because there are more objects in the struct between `write_queue` and `pid_t`. Look at #427 for a guide to how you need to get this working.

Reply:
Maybe we should add this to the walkthrough as it will be a frequent problem.

Reply:
Agreed.